carbon-design-system/carbon-website
# Update TypesetExample.js to use em instead of rem

Closes https://github.com/carbon-design-system/carbon/issues/6545

We updated our type implementation to use the relative `em` value versus `rem` for values like line-height. This PR updates the TypesetExample component to use `em` instead of `rem` for the unit for line-height.

#### Changelog

**New**

**Changed**

- Update `TypesetExample` to use `em` instead of `rem` for line-height values

**Removed**

#### Testing and reviewing

- Visit the productive typography page in the preview link
- Verify that line-height values are expressed in `em` values instead of `rem`
